### DISCUSSION TITLE
Communication between devices

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#145] [Q&A] [answered]
In my opinion, everything that can make the routing algorithm similar to the Kurviger is an argument for switching from it to the Cruiser. I am closely watching the development of both applications, because the situation in which I set routes in one and navigate in the other is very uncomfortable.

I love Cruiser's UI and logic, but the inability (or just only my inability?) to route secondary and tertiary local roads keeps me from paying the subscription fee (which I probably will sooner or later anyway) :)

_Originally posted by @Ciutnik in https://github.com/devemux86/cruiser/discussions/138#discussioncomment-5759052_

**Comment by devemux86** [comment, marked as answer]
**Free cloud providers** are still the best choice for communication between devices: Google Drive, Dropbox, ...

_Advantages_:
- _Offline use_
- Sub folders
- Store extra files, favorites, gpx
- Sort as you like
- Share with other users
- _User privacy_

---

- https://forum.kurviger.de/t/access-web-route-from-app/4925/9

**Reply by Ciutnik** [reply]
Thanks for splitting the topic. Of course, the cloud is the easiest way to transfer files and I am fully aware of it. 

Or would it be possible to simplify and automation opening of freshly synced files in Cruiser? If the application could monitor the selected folder, and after its launch, if new supported files (*.kurviger, *.gpx) were detected, suggest its import?

**Comment by devemux86** [comment]
> Or would it be possible to simplify and automation opening of freshly synced files in Cruiser? If the application could monitor the selected folder, and after its launch, if new supported files (*.kurviger, *.gpx) were detected, suggest its import?

@Ciutnik thanks for the idea!

This could be seen as part of the next features that are in the process of being implemented:

- https://github.com/devemux86/cruiser/discussions/35
- https://github.com/devemux86/cruiser/discussions/81
